src/qibolab/instruments/qm/program/instructions.py

```diff
@@ -20,15 +20,15 @@ def _delay(pulse: Delay, element: str, parameters: Parameters):
     if parameters.duration is None:
         duration = int(pulse.duration) // 4
     else:
-        duration = parameters.duration
+        duration = parameters.duration / 4
```

Comment:
```suggestion
        duration = parameters.duration // 4
```
don't you need an integer even in this case? (`/` is float division even between two integers, so `//` will give you more confidence the result is what you expect - if you expected an `int`)